watertap/property_models/seawater_prop_pack.py

```diff
@@ -625,7 +625,7 @@ def build(self):
         self.bpe_A0 = Var(
             within=Reals,
             initialize=17.95,
-            units=t_inv_units,
+            units=pyunits.K,
```

Comment:
Whilst it doesn't really matter here, it is generally good practice to take the units from the property package metadata rather than hard-coding it here.

Note also that IDAES recently changed how it handles metadata for units - whilst the old approach still works, the new (improved) API is:

```
units_metadata = self.get_metadata().default_units

temperature_units = units_metadata.TEMPERATURE
```

Note this this should support autocompletion in most IDEs.

Reply:
Thanks a lot for the detailed suggestions, Andrew. One quick question regarding the metadata of units, currently the seawater property package uses the old approach to specify the units for all other property variables, do you think I should update them as well?

Reply:
It would be nice, but hardly necessary. So, if you have the time it would be good to do (and serve as an example of how to do things in the future), but if there are more pressing things to do it should not hold up this PR.

Reply:
Thank you, it makes sense.